axop/plugins.py

```diff
@@ -34,6 +35,34 @@ class InternalPlugin(): # pylint: disable=too-few-public-methods
     def __init__(self, kplugin: PluginKind):
         self._kplugin = kplugin
 
+    def decode(self, input_str: str) -> str:
+        """
+        Decode base64 and unzip
+        """
+        base64_bytes = input_str.encode('utf-8')
+        message_bytes = base64.b64decode(base64_bytes)
+        uncmpstr = zlib.decompress(message_bytes)
```

Comment:
can trigger an exception if input_str is not base64 encoded (ex: input_str = "test")

Reply:
true